### PR TITLE
dev access key in the "install-web-dir" command

### PR DESCRIPTION
**Pull request by aschempp**
Removes storing the dev access key in the `install-web-dir` command, where it actually never should have been. In the previous implementation the access key was stored multiple times (in manager config and in `.env` file). It also meant that removing the key through the command would re-add it the next time the command is run without arguments (because the manager config still exists). This is now all fixes by simply updating the `.env` file through the API.

### CODE REVIEW

File: src/Api/Application.php [hunk 1]
```diff
@@ -100,6 +111,7 @@ protected function getDefaultCommands(): array
         $commands[] = new VersionCommand();
         $commands[] = new GetConfigCommand();
         $commands[] = new SetConfigCommand();
+        $commands[] = new DebugAccesskeyCommand();
```
Comment (leofeyer) [inline src/Api/Application.php]: Why are you not passing `$this->projectDir` in the constructor?

Comment (aschempp) [reply src/Api/Application.php]: Changed in 0b49f9a3a1b0f5c9ba27b01644c82bd551dfe76d (for all commands actually)

File: src/Api/Application.php [hunk 2]
```diff
@@ -45,6 +46,16 @@ public function __construct(string $projectDir)
         parent::__construct('contao-api', self::VERSION);
     }
 
+    /**
+     * Gets the project root directory.
+     *
+     * @return string
+     */
+    public function getProjectDir(): string
```
Comment (leofeyer) [inline src/Api/Application.php]: You could remove the method now that you are injecting the project dir.

**Comment by leofeyer** [comment]
Please revert the changes in the `InstallWebDirCommand.php` file. They are not related to the dev access key changes and I cannot merge this PR with them in.

File: src/Api/Command/DebugAccesskeyCommand.php [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure(): void
+    {
+        parent::configure();
+
+        $this
+            ->setName('debug:access-key')
```
Comment (leofeyer) [inline src/Api/Command/DebugAccesskeyCommand.php]: The namespace is misleading. Every other command starting with `debug:` prints debug information so I would expect a `debug:access-key` to print my access key configuration and not to set or remove the access key.

Comment (leofeyer) [reply src/Api/Command/DebugAccesskeyCommand.php]: What about `access-key:set` and `access-key:remove`?

File: src/HttpKernel/ContaoKernel.php [hunk 4]
```diff
@@ -108,15 +108,7 @@ public function getLogDir(): string
...
-            $this->pluginLoader = new PluginLoader($this->getProjectDir().'/vendor/composer/installed.json');
-
-            $config = $this->getManagerConfig()->all();
-
-            if (isset($config['contao_manager']['disabled_packages'])
-                && \is_array($config['contao_manager']['disabled_packages'])
-            ) {
-                $this->pluginLoader->setDisabledPackages($config['contao_manager']['disabled_packages']);
-            }
+            $this->pluginLoader = new PluginLoader();
```
Comment (leofeyer) [inline src/HttpKernel/ContaoKernel.php]: Please revert these changes. They are not related to the dev access key changes and I cannot merge this PR with them in.

File: src/Api/Command/DebugAccesskeyCommand.php (hunk 3 above)
Comment (aschempp) [reply src/Api/Command/DebugAccesskeyCommand.php]: Good point, changed in 14c921bdc99d960fa110c703b6db72a98d580dc6

**Comment by aschempp** [comment]
The changes in `ContaoKernel` are not unrelated, they update the `manager-bundle` to match the `manager-plugin`. The original title and description never sayed this is only to add new API commands 😉

**Comment by leofeyer** [comment]
Thank you @aschempp.
